frontend/components/markup-modal.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'react-modal';
+import ClickToSelect from '@mapbox/react-click-to-select';
+import { resolveUri } from './badge-examples';
+
+export default class MarkupModal extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
```

Comment:
You can use [class properties](https://babeljs.io/docs/plugins/transform-class-properties/) to set this, rather than doing it in the constructor:
```
export default class MarkupModal extends React.Component {
  state = {
    // ...
  };
}
```

Note: This requires the appropriate Babel preset to be enabled (it should be)